wpilibcIntegrationTests/src/CircularBufferTest.cpp

```diff
@@ -5,12 +5,11 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "CircularBuffer.h"
+#include <array>
 
+#include "CircularBuffer.h"
```

Comment:
For tests I think we want to include the tested header first.  For header only implementations the unit test may be the only place guaranteed to include it first.